test/unit/test_resubmit.js

```diff
@@ -0,0 +1,62 @@
+describe("ResubmitController", function () {
+
+    beforeEach(module("SquadResubmit"));
+
+    var $controller;
+
+    beforeEach(inject(function(_$controller_){
+        $controller = _$controller_;
+    }));
+
+    describe("$scope.resubmit", function () {
+
+        var $httpBackend;
+
+        beforeEach(function() {
+            $scope = {};
+            $attrs = {};
+            $location = "";
+            controller = $controller('ResubmitController', {
+                $scope: $scope,
+                $attrs: $attrs,
+                $location: $location
+            });
+        });
+
+        beforeEach(inject(function($injector) {
+            $timeout = $injector.get('$timeout');
+            $httpBackend = $injector.get('$httpBackend');
+            $httpBackend.whenGET("/api/resubmit/1").respond(
+                200, ["submitted"]);
+            $httpBackend.whenGET("/api/forceresubmit/1").respond(
+                401, ["error"]);
+        }));
+
+        afterEach(function() {
+            $httpBackend.verifyNoOutstandingExpectation();
+            $httpBackend.verifyNoOutstandingRequest();
+        });
+
+        it('tests resubmit function when response is OK', function () {
+            $scope.resubmit(1, false)
+            $httpBackend.flush();
+
+            $timeout.flush();
+            $timeout.verifyNoPendingTasks();
+
```

Comment:
Perhaps there could be `expect($scope.error).toBe(false)`, since the spec below does `expect($scope.error).toBe(true)`